recipes/biorbd-casadi/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+mkdir build
+cd build
+
+cmake .. \
+      -GNinja \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DMATH_LIBRARY_BACKEND="Casadi" \
+      -DBUILD_EXAMPLE=OFF \
+      -DMODULE_ACTUATORS=ON \
+      -DMODULE_MUSCLES=ON \
+      -DBINDER_PYTHON3=ON \
+        -DNUMPY_INCLUDES=$PREFIX/numpy/core/include \
```

Comment:
Is this location correct?

Reply:
Assuming anaconda, it is correct

Reply:
That looks indeed weird, I'll try straight up remove this since NUMPY_INCLUDES is supposed to be found automatically anyway

Reply:
Done